src/main/resources/database/data.sql

```diff
@@ -141,17 +145,32 @@ INSERT INTO `shift_comment_position` (comment_id, position_id) VALUES (33, 1);
 INSERT INTO `shift_comment_position` (comment_id, position_id) VALUES (34, 1);
 INSERT INTO `shift_comment_position` (comment_id, position_id) VALUES (35, 1);
 
-INSERT INTO `location` (l_title, l_info) VALUES ('restaurant 1', 'restaurant in hotel');
-INSERT INTO `location` (l_title, l_info) VALUES ('restaurant 2', 'restaurant outside');
-INSERT INTO `location` (l_title, l_info) VALUES ('laundry', 'laundry');
-INSERT INTO `location` (l_title) VALUES ('room 200');
-
-INSERT INTO `task` (u_id_assignee, u_id_delegator, t_message, t_status, priority, t_frequency, updated) VALUES (1, 2, 'clean rooms', @OPENED, 1, 'DAILY', CURRENT_TIMESTAMP);
-INSERT INTO `task` (u_id_assignee, u_id_delegator, t_message, t_status, priority, t_frequency, updated) VALUES (2, 2, 'call taxi', 'IN_PROGRESS', 2, 'WEEKLY', CURRENT_TIMESTAMP);
-INSERT INTO `task` (u_id_assignee, u_id_delegator, t_message, t_status, priority, t_frequency, updated) VALUES (3, 3, 'look in the window', @CLOSED, 5, 'MONTHLY', CURRENT_TIMESTAMP);
+INSERT INTO `location` (l_title) VALUES ('Мальвы');
+INSERT INTO `location` (l_title) VALUES ('ANISE');
+INSERT INTO `location` (l_title) VALUES ('Отель');
+INSERT INTO `location` (l_title) VALUES ('Кухня');
+INSERT INTO `location` (l_title) VALUES ('1 этаж');
+INSERT INTO `location` (l_title) VALUES ('2 этаж');
+INSERT INTO `location` (l_title) VALUES ('3 этаж');
+INSERT INTO `location` (l_title) VALUES ('4 этаж');
+INSERT INTO `location` (l_title) VALUES ('5 этаж');
+
+INSERT INTO `task` (u_id_assignee, u_id_delegator, t_message, t_status, priority, t_frequency, updated, expired) VALUES (1, 2, 'clean rooms', @OPENED, 2, 'DAILY', CURRENT_TIMESTAMP, TIMESTAMPADD(DAY, 1, CURRENT_TIMESTAMP));
+INSERT INTO `task` (u_id_assignee, u_id_delegator, t_message, t_status, priority, t_frequency, updated, expired) VALUES (5, 2, 'call taxi', @OPENED, 3, 'WEEKLY', CURRENT_TIMESTAMP, TIMESTAMPADD(DAY, 2, CURRENT_TIMESTAMP));
+INSERT INTO `task` (u_id_assignee, u_id_delegator, t_message, t_status, priority, t_frequency, updated, expired) VALUES (3, 3, 'look in the window', @CLOSED, 4, 'MONTHLY', CURRENT_TIMESTAMP, TIMESTAMPADD(DAY, 1, CURRENT_TIMESTAMP));
+INSERT INTO `task` (u_id_assignee, u_id_delegator, t_message, t_status, priority, t_frequency, updated, expired) VALUES (2, 1, 'clean some room', @OPENED, 1, 'ONCE', CURRENT_TIMESTAMP, TIMESTAMPADD(DAY, 4, CURRENT_TIMESTAMP));
+INSERT INTO `task` (u_id_assignee, u_id_delegator, t_message, t_status, priority, t_frequency, updated, expired) VALUES (5, 1, 'close all doors', @OPENED, 5, 'ONCE', CURRENT_TIMESTAMP, TIMESTAMPADD(DAY, 2, CURRENT_TIMESTAMP));
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal 3 times. ([plsql:S1192](https://sonarcloud.io/organizations/final-project/rules#rule_key=plsql%3AS1192))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=125&issues=AWYWGDx8vOWp7Yd6qIeh&open=AWYWGDx8vOWp7Yd6qIeh)

[ISSUE_KEY]: AWYWGDx8vOWp7Yd6qIeh